ckan/logic/action/get.py

```diff
@@ -3047,6 +3047,9 @@ def _followee_count(context, data_dict, FollowerClass):
         data_dict, errors = _validate(data_dict, schema, context)
         if errors:
             raise ValidationError(errors)
+    if data_dict.get('__extras'):
```

Comment:
We should never rely on `__extras` as these are just byproducts of the validation process. It's better if whatever function that is calling `_followee_count` passes an explicit `is_organization` parameter (that defaults to False)